README.md

```diff
@@ -50,123 +48,32 @@ That will require the following dependencies (`slf4j-log4j12` forwards logs to t
- 1. Find the name of the Kinesis stream you are using to send data to ELK
- 1. If the kinesis stream is not owned by your target account, find the name of the role used to permit enqueing to that stream
-
-Target account actions:
-
- 1. The name of this bucket needs to be available in SSM under the key `/account/services/artifact.bucket`
- 1. The ARN of the target kinesis stream needs to be available in SSM under the key `/account/services/logging.stream`
-
-This project actions:
-
- 1. Add your stack name to the `stacks` section of `riff-raff.yaml`
- 1. Deploy this project - this will deploy [a new CloudFormation stack](./template.yaml) to your account
- 1. If you want to change the retention or need to add a role to assume when writing to the Kinesis stream you can manually change the settings in the stack after this initial deploy
-
-### Riff-Raff permissions
-
-Your riff-raff user will required the following permissions:
+## Deployment
+> **Note**
+> This stack is maintained by [@guardian/devx-operations](https://github.com/orgs/guardian/teams/devx-operations), who would be happy to help with any issues.
 
- - s3:*
- - iam:*
- - cloudformation:*
- - lambda:*
- - events:DescribeRule
- - events:PutRule
- - ssm:GetParameter*
+This service is deployed via Riff-Raff, see `tools::cloudwatch-logs-management`.
 
-This can be simply added as a single policy:
-
-```
-{
-    "Version": "2012-10-17",
-    "Statement": [
-        {
-            "Sid": "CloudwatchLogsManagement",
-            "Effect": "Allow",
-            "Action": [
-                "s3:*",
-                "iam:*",
-                "cloudformation:*",
-                "lambda:*",
-                "events:DescribeRule",
-                "events:PutRule",
-                "ssm:GetParameter*"
-            ],
-            "Resource": "*"
-        }
-    ]
-}
-```
-
-and attached to the riffraff user with the following command:
-
-```
-aws --profile $PROFILE --region $REGION iam attach-user-policy --user-name riffraff --policy-arn $POLICY_ARN 
-```
-
-### Bucket Parameter
-
-List buckets:
-```
-aws --profile $PROFILE --region $REGION s3 ls
-```
-
-Create parameter:
-```
-aws --profile $PROFILE --region $REGION ssm put-parameter --name '/account/services/artifact.bucket' --value $VALUE --type String
-```
-
-# Kinesis parameter:
-
-Assuming your elk kinesis stream is in the same account, you can use the following commands.  If not, ask which ARN you should use for
-the third command below.
-
-List streams:
-```
-aws --profile $PROFILE --region $REGION kinesis list-streams
-```
-
-Get stream ARN:
-```
-aws --profile $PROFILE --region $REGION kinesis describe-stream --stream-name $NAME | jq -r .StreamDescription.StreamARN
-```
-
-Create parameter:
-```
-aws --profile $PROFILE --region $REGION ssm put-parameter --name '/account/services/logging.stream' --value $VALUE --type String
-```
```

Comment:
Swap this for "contact the team if you need help" as these steps go against the "everything in VCS" idea.